docs/octopus-rest-api/octopus-cli/index.md

```diff
@@ -27,7 +27,7 @@ hideInThisSection: true
 - **[deploy-release](/docs/octopus-rest-api/octopus-cli/deploy-release.md)**:  Deploys a release.
 - **[dump-deployments](/docs/octopus-rest-api/octopus-cli/dump-deployments.md)**:  Writes deployments to an XML file that can be imported in Excel.
 - **[export](/docs/octopus-rest-api/octopus-cli/export.md)**:  Exports an object to a JSON file. Deprecated. Please see [https://g.octopushq.com/DataMigration](https://g.octopushq.com/DataMigration) for alternative options.
-- **[import](/docs/octopus-rest-api/octopus-cli/import.md)**:  Imports an Octopus object from an export file. Deprecated. Please see [Data Migration](https://g.octopushq.com/DataMigration) for alternative options.
+- **[import](/docs/octopus-rest-api/octopus-cli/import.md)**:  Imports an Octopus object from an export file. Deprecated. Please see [https://g.octopushq.com/DataMigration](https://g.octopushq.com/DataMigration) for alternative options.
```

Comment:
@matt-richardson, I can't figure out where this text is generated. Does it come from the OctopusDeploy repo, or is this generated by the script that pushes changes to these PRs?

Reply:
It comes from https://github.com/OctopusDeploy/OctopusCLI/blob/master/source/Octopus.Cli/Commands/ImportCommand.cs#L35. I _think_ that the doc generator automatically creates links when it hits a url, but at that point, we dont have any link text for it.